### PR TITLE
update Featuretools v0.6.0 prerelease

### PR DESCRIPTION
**Pull request by bukosabino**

### CODE REVIEW

**Comment by kmax12** [comment]
@bukosabino looks like `no_show` is being leaked into the model. it is getting 1.0 AUC. can you double check the notebook?

**Comment by kmax12** [comment]
@bukosabino can you report what there error was the required you to pin the pandas version? we should look into fixing it.

**Comment by bukosabino** [comment]
> @bukosabino can you report what there error was the required you to pin the pandas version? we should look into fixing it.

pandas==0.24.0
TypeError: Cannot compare tz-naive and tz-aware datetime-like objects
When you run ft.dfs()

File: requirements.txt [hunk 1]
```diff
@@ -2,3 +2,4 @@ featuretools>=0.5.1
 scikit-learn>=0.20.1
 bokeh>=1.0.2
 jupyter>=1.0.0
+pandas==0.23.4
```
Comment (kmax12) [inline requirements.txt]: We pinned pandas to 0.23.4 in the latest release of featuretools, no need to add it to the requirements

**Pull request merged**